CODING_GUIDELINES.md

```diff
@@ -211,44 +217,6 @@ a += "b"
   + "d";
 ```
 
-* Initializers
-
-We allow brace enclosed initializers only for aggregates and arrays/containers.<br />
-Brace enclosed initializer MUST be used with equality sign if it follows the variable declaration.<br />
-Brace enclosed initializer MUST be additionally enclosed in parentheses if it is used in constructor initialization list.<br />
-Some valid use cases:
-```c++
-// aggregate
-Person john = { "John", "Smith", 21 };
-Person *john = new Person { "John", "Smith", 21 };
-
-// array
-int array[] = { 1, 2, 3, 4 };
-
-// container
-QHash<QString, QString> map = {
-    { "key1", "value1" },
-    { "key2", "value2" }
-);
-
-// member array
-SomeClass::SomeClass(BaseClass *parent)
-    : BaseClass(parent)
-    , m_someArrayMember({ 1, 2, 3, 4 })
-{
-}
-
-// return from function
-Person getPersonByName(const QString &name)
-{
-    // do something
-    return { name, surname, age };
-}
-
-// function argument
-doSomething({ name, surname, age }, someOtherArg);
-```
-
 * **auto** keyword
 
 We allow the use of the **auto** keyword only where it doesn't break the readability of the code (i.e. either we can gather enough information about the type from the right part of the expression, or we do not need to know the exact type), or where it is strictly necessary (for example, to compute the type of a lambda, etc.).<br />
```

Comment:
> Yes. For example, checking someone's PR here (on GitHub).

This adds another valuable bit of justification. Why don't you add a couple of sentences on this subject then? Something like the following:

When weighting whether to use an auto-typed variable please think about potentials reviewers of your code, who will read it as a plain diff (on github.com, for example). Please make sure that such reviewers can understand the code completely and without excessive efforts.